src/image/makeWcs.cc

```diff
@@ -62,12 +62,9 @@ afwImg::Wcs::Ptr afwImg::makeWcs(
     //
     // Follow Dave's AST and switch TAN to TPV
     //
-    using pex::logging::Log;
-    auto log = Log(Log::getDefaultLog(), "makeWcs");
-
     if (ctype1.substr(5, 3) == "TAN" &&
         (metadata->exists("PV1_5") || metadata->exists("PV2_1"))) {
-        log.log(Log::INFO, str(boost::format("Interpreting %s/%s + PVi_j as TPV") % ctype1 % ctype2));
+        LOGLS_INFO("makeWcs", "Interpreting " << ctype1 << "/" << ctype2 << " + PVi_j as TPV");
```

Comment:
I think `LOGL_DEBUG` with `sprintf`-style formatting would make this a bit more concise.